extensions/extension_process/xwalk_extension_process.cc

```diff
@@ -124,7 +124,7 @@ bool XWalkExtensionProcess::CheckAPIAccessControl(
   PermissionCacheType::iterator iter =
       permission_cache_.find(extension_name + api_name);
   if (iter != permission_cache_.end())
-    return iter->second;
+    return iter->second != ALLOW_ONCE;
```

Comment:
This change is from https://github.com/crosswalk-project/crosswalk/commit/231ec374c53fed0166357aa4e013c8b246abaddd